test/support/src/vfs_helpers.cc

```diff
@@ -173,6 +173,10 @@ Status SupportedFsS3::prepare_config(
     [[maybe_unused]] tiledb_config_t* config,
     [[maybe_unused]] tiledb_error_t* error) {
 #ifndef TILEDB_TESTS_AWS_S3_CONFIG
+  if (rest_) {
```

Comment:
Could we set `tiledb_tests_aws_s3_config` in build-libtiledb.sh on the internal repository when we bootstrap the core build? Either way SGTM so approved, but the ifndef above seems like it could accomplish this

Reply:
We could, but it is misleading, since we don't actually test on Amazon S3. I would like us eventually to entirely remove VFS configuration from test code, and rely on environment variables.